lib/fog/cloudsigma/models/snapshot.rb

```diff
@@ -0,0 +1,76 @@
+require 'fog/cloudsigma/nested_model'
+require 'fog/cloudsigma/models/volume'
+
+module Fog
+  module Compute
+    class CloudSigma
+      class Snapshot < Fog::CloudSigma::CloudsigmaModel
+        identity :uuid
+
+        attribute :timestamp, :type => :string
+        attribute :tags
+        attribute :status, :type => :string
+        attribute :resource_uri, :type => :string
+        attribute :permissions, :type => :array
+        attribute :owner
+        attribute :name, :type => :string
+        attribute :meta
+        attribute :grantees, :type => :array
+        attribute :drive
+        attribute :allocated_size, :type => :integer
```

Comment:
Ok, sure. Should I do the same in tests/cloudsigma/requests/*_tests.rb where we specify reply format?